tools/kbump/pkg/kbump/kbump.go

```diff
@@ -0,0 +1,82 @@
+package kbump
+
+import (
+	"errors"
+	"sigs.k8s.io/yaml"
+
+	"github.com/aws/karpenter/pkg/apis/v1alpha1"
+	nodeclassutil "github.com/aws/karpenter/pkg/utils/nodeclass"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
+	nodepoolutil "github.com/aws/karpenter-core/pkg/utils/nodepool"
+)
+
+type Parameters struct {
+	Role *string
+}
+
+func Process(inputYAML []byte, params *Parameters) ([]byte, error) {
+	var input metav1.TypeMeta
+	err := yaml.Unmarshal(inputYAML, &input)
+	if err != nil {
```

Comment:
If you have errors like this and you don't need to retain their output. General practice is that we can collapse these down into a single line

```go
if err := yaml.Unmarshal(inputYAML, &input); err != nil 
```